ci/docker/Dockerfile.build.jetson

```diff
@@ -37,6 +37,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     unzip \
     python3 \
     python3-pip \
+    python \
+    python-pip \
```

Comment:
Do we need both pip and pip3?

Reply:
pip3 existed beforehand. I need python2 since build_wheel uses python2